### ISSUE TITLE
TypeError: Cannot read property 'replace' of undefined

### ISSUE THREAD

**Issue by MadeInChina** [webpro/reveal-md#196]
download this md file and run  reveal-md README.md --static _site
https://github.com/theodorosploumis/docker-presentation/blob/gh-pages/readme.md
reveal-md  -V
2.0.4
```
 reveal-md README.md --static _site
ImageDataURI :: Error :: {
    "code": "UNABLE_TO_VERIFY_LEAF_SIGNATURE"
}
TypeError: Cannot read property 'replace' of undefined
    at parseYamlFrontMatter (/usr/local/lib/node_modules/reveal-md/lib/render.js:14:54)
    at parseSlides (/usr/local/lib/node_modules/reveal-md/lib/render.js:26:16)
    at render (/usr/local/lib/node_modules/reveal-md/lib/render.js:37:10)
    at parseSlideAwait.then.then.markdown (/usr/local/lib/node_modules/reveal-md/lib/static.js:51:23)
    at tryCatcher (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/usr/local/lib/node_modules/reveal-md/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:763:18)
    at tryOnImmediate (timers.js:734:5)
    at processImmediate (timers.js:716:5)
```

**Comment by MadeInChina** [comment]
issue with this link.
```
![Docker vs traditional Virtualization](https://insights.sei.cmu.edu/assets/content/VM-Diagram.png)
```

**Comment by webpro** [comment]
Some images don't make it through the image encoder, not sure why.

Anyway, I've just released [v2.0.5](https://github.com/webpro/reveal-md/releases/tag/2.0.5) which logs errors, but continues rendering.
